scripts/release-android.sh

```diff
@@ -61,6 +61,11 @@ if [[ -n "${READER_FEATURES}" ]]; then
     append_env_export 'READER_FEATURES'
 fi
 
+# Used by android/build.gradle to make dependencies/plugins optional
+if [[ -n "${GOOGLE_FREE}" ]]; then
+    append_env_export 'GOOGLE_FREE'
+fi
```

Comment:
Yeah, this is a good catch. Since we add `GOOGLE_FREE=1` to `.env.release`, but that doesn't affect Gradle in Nix.

But I really need to get around to refactoring one of these days how these settings are passed via env. This is unnecessarily complex. It grew out of just a few variables, and it needs to be simplified.